Python/pystate.c

```diff
@@ -294,6 +294,8 @@ PyInterpreterState_Clear(PyInterpreterState *interp)
     Py_CLEAR(interp->codec_error_registry);
     Py_CLEAR(interp->modules);
     Py_CLEAR(interp->modules_by_index);
+    PyDict_Clear(interp->sysdict);
+    PyDict_Clear(interp->builtins);
     Py_CLEAR(interp->sysdict);
     Py_CLEAR(interp->builtins);
```

Comment:
You should move these lines to the end of the function (before the XXX comment). We discussed that in a previous comment. And I asked you to add a comment explaining why we have to clear the dict at the end of the function. I don't understand why you removed the comment.

Reply:
Oh, sorry. I just force your code, not mention your description info~.

Reply:
done.